plotsearch/models.py

```diff
@@ -172,6 +173,31 @@ def clean(self):
         raise ValidationError(code="no_adding_searchable_targets_after_begins_at")
 
 
+class InformationCheck(NameModel):
```

Comment:
I suppose the name field inherited from NameModel is intended to contain the enum-like value that signifies what piece of information this is? If so, then this model seems  okay for the needs of the applicant section at least.

Reply:
NameModel defines abstract model which has required charfield called name and also str(NameModel) returns name

Reply:
Yeah, I read the definition of NameModel while reviewing this; I was just wondering about where that type-of-check data would go and presumed this was the only spot it could fit.

I guess I could reword the question as so: if we were to have all InformationChecks for kaupparekisteriote/trade register extract contain that type as "trade_register_extract" in some field, would it be this one or do we need a new one for that?

Reply:
I was thinking that this would be the field. There is also possibility to inherit model straight from Models-class. Then we could define name which would be some sort of EnumField. But I think this will do fine and there are no restrictions if we want to change names.

Reply:
Well now thinking this again, there could be some translation benefits if they are enums

Reply:
I guess so. Then again, these fields are only shown on the administrative UI which is only presented in Finnish, so the other-language translations might not appear anywhere even if present.